api.go

```diff
@@ -315,6 +316,11 @@ type (
 		CatalystPipelineStrategy string    `json:"catalystPipelineStrategy,omitempty"`
 	}
 
+	DeleteAssetTaskParams struct {
+		Id            string `json:"id"`
+		ObjectStoreId string `json:"objectStoreId"`
```

Comment:
You don't need this! The task should already have an `inputAssetId` field, which `task/runner.go` will already make sure to fetch the full object, grab the object store id from that object, fetch the object store and instantiate the client for your task handler. Check the task context

Reply:
Yeah I noticed when editing the task runner! Will update that :)